### DISCUSSION TITLE
Notifications anti-patterns and hard-to-parse inboxes

### DISCUSSION THREAD

**Discussion by tkoft** [community/community#116671] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The main notifications view on desktop is pretty hard to parse for me. Looking at this, I don't actually get any sense of what's happened that triggered each notification, especially when multiple updates are stacked together. Clicking through sometimes will deep link me to a mention if that's the only update that's being notified, but a lot of the times I feel like I just have to parse the whole PR discussion again and try to figure out what the new comments are that are relevant to me.
<img width="1061" alt="Screenshot 2024-04-03 at 12 10 03 PM" src="https://github.com/community/community/assets/14863378/8bc5307d-6f4c-4e81-abe8-ca59911b06c1">

The root issue here to me is that these notifications are focused on the entity acted on rather than the action. The github mobile app does a better job of actually telling you what happened, but it's secondary text and truncated to the point of almost being entirely useless.
<img width="300" src="https://github.com/community/community/assets/14863378/424a9d05-85f4-4773-9440-96dd0bbb21f2">

Compare this to facebook notifications, or any other social app. Clear and actionable. "X did Y to Z". 
<img width="300" src="https://github.com/community/community/assets/14863378/17c5d54b-bba0-49d4-b06a-1dae48d10fb5">

**Comment by joakimtall** [comment]
100% agree, I get confused what has happened every time I click a notification on a bigger PR. Add some way of expanding the notification to see all of them, and show something more then "mention"
